src/components/DetailsArtifacts/DetailsArtifacts.js

```diff
@@ -116,10 +117,27 @@ const DetailsArtifacts = ({
     )
   }
 
+  const showArtifact = useCallback(
+    index => {
+      const newArtifactsIndexes = artifactsIndexes.filter(
+        artifactIndex => artifactIndex !== index
+      )
+
+      if (artifactsIndexes.indexOf(index) === -1) {
```

Comment:
Please use `.includes()` with `!` instead of `.indexOf() === -1`
```suggestion
      if (!artifactsIndexes.includes(index)) {
```

Reply:
done